src/vision.md

```diff
@@ -0,0 +1,26 @@
+# 🔮 The vision
+
+[cc]: ./vision/characters.md
+[Grace]: ./vision/characters/grace.md
+[Alan]: ./vision/characters/alan.md
+[gba]: ./vision/characters/grace.md#variant-a-networking-systems
+[gba]: ./vision/characters/grace.md#variant-b-embedded
+[sq]: ./vision/status_quo.md
+[gsq]: ./vision/status_quo/gracy_deploys_her_service.md
+[cok]: https://en.wikipedia.org/wiki/Curse_of_knowledge
+[sf]: ./vision/shiny_future.md
+[roadmap]: ./vision/roadmap.md
+
+## What is this
+
+We believe Rust can become one of the most popular choices for building distributed systems, ranging from embedded devices to foundational cloud services. Whatever they're using it for, we want all developers to love using macros Rust.
```

Comment:
I think the distributed systems sentence is probably more specific to async than macros, since macros aren't as focused on distributed systems. It'd be good to focus on why people love or want to use macros specifically.

Reply:
I reused the sentences, but currently, I have no version for macros, I was thinking of removing it for the moment, what do you think?